engine/src/main/java/org/hibernate/search/engine/mapper/mapping/building/spi/IndexedEntityBindingContextProvider.java

```diff
@@ -8,9 +8,15 @@
 
 import java.util.Optional;
 
-public interface IndexManagerBuildingStateProvider {
+import org.hibernate.search.engine.backend.document.model.dsl.spi.IndexSchemaRootNodeBuilder;
+
+public interface IndexedEntityBindingContextProvider {
```

Comment:
This is continuing to be a provider of `IndexManagerBuildingState`, why shouldn't it be reflected in its name? But I think that the naming problem here is the reflex of a design issue. See the other comment.